aspnetcore/signalr/background-service/sample/Clients.ConsoleOne/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading.Tasks;
+using HubServiceInterfaces;
+using Microsoft.AspNetCore.SignalR.Client;
+
+namespace Clients.ConsoleOne
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var connection = new HubConnectionBuilder()
+                .WithUrl(Strings.HubUrl)
+                .Build();
+
+            connection.On<DateTime>(Strings.Events.TimeSent, (dateTime) =>
+            {
+                Console.WriteLine(dateTime.ToString());
+            });
+
+            while (true)
```

Comment:
Maybe add a comment here describing the loop here is just to make sure the server is running.